src/components/filters-history/FiltersHistoryItem.tsx

```diff
@@ -49,7 +49,31 @@ const FiltersHistoryItem = ({ item, filter }: Props) => {
 		<div
 			className='filters-history__item'
 			onClick={() => {
-				filter.setState(item.filters as Partial<EventFilterState> & Partial<MessageFilterState>);
+				const toReset = Object.fromEntries(
+					Object.entries(filter.state)
+						.filter(([key]) => !Object.keys(item.filters).includes(key))
```

Comment:
there is a `getDefaultEventsFiltersState` and 'getDefaultMessagesFiltersState' functions, get a default filter and spread over new changes